src/main/java/com/launchdarkly/client/FeatureFlagBuilder.java

```diff
@@ -0,0 +1,83 @@
+package com.launchdarkly.client;
+
+import com.google.gson.JsonElement;
+
+import java.util.List;
+
+public class FeatureFlagBuilder {
```

Comment:
I don't think this should be public-- we don't expect clients to be constructing `FeatureFlag`s